invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -12,45 +12,65 @@
 <div class="ui container fluid page-subheader-outer with-submenu rel-pt-2">
   <div class="ui container relaxed grid page-subheader">
     <div class="row pb-0">
-      <div class="sixteen wide mobile five wide tablet two wide computer column">
-        <div class="ui rounded fluid image community-image rel-mb-2">
+      <div class="sixteen wide mobile four wide tablet two wide computer column rel-mb-1">
+        <div class="ui rounded fluid image community-image">
           <img src="{{ community.links.logo }}" alt="Community logo" onerror="this.onerror=null;this.src='/static/images/square-placeholder.png';"/>
         </div>
       </div>
 
-      <div class="sixteen wide mobile eleven wide tablet eleven wide computer column rel-mb-2">
+      <div class="sixteen wide mobile twelve wide tablet eleven wide computer column rel-mb-2">
         <h2 class="ui header">{{ community.metadata.title }}</h2>
-        <div class="rel-mb-2">
-          {{ truncate_text(community.metadata.description) }}
-        </div>
 
-        {% if community.metadata.type is defined %}
-        <div class="mt-5">
-          <i class="tag icon" aria-hidden="true"></i>
-          <div class="ui label small grey type-label">
-            {{ community.metadata.type.title.en }}
+        <div>
+          {% if community.metadata.website %}
+          <div class="inline-computer mt-5 rel-mr-1">
+            <i class="linkify icon" aria-hidden="true"></i>
+            <a href="{{community.metadata.website}}">
+              {{community.metadata.website}}
+            </a>
           </div>
-        </div>
-        {% endif %}
+          {% endif %}
 
-        {% if community.metadata.website %}
-        <div class="mt-5">
-          <i class="linkify icon" aria-hidden="true"></i>
-          <a href="{{community.metadata.website}}">
-            {{community.metadata.website}}
-          </a>
-        </div>
-        {% endif %}
+          {% if community.metadata.type is defined %}
+          <div class="inline-computer mt-5 rel-mr-1">
+            <i class="tag icon" aria-hidden="true"></i>
+            <span class="label label-keyword">
+              {{ community.metadata.type.title.en }}
+            </span>
+          </div>
+          {% endif %}
 
-        {% if community.metadata.organizations %}
-        <div class="mt-5">
-          <i class="building outline icon" aria-hidden="true"></i>
-          <span class="label label-keyword">{{ community.metadata.organizations| map(attribute='name') | join(', ') }}</span>
+          {% if community.metadata.organizations %}
+          <div class="inline-computer mt-5 rel-mr-1">
+            <i class="building outline icon" aria-hidden="true"></i>
+            <span class="label label-keyword">{{ community.metadata.organizations| map(attribute='name') | join(', ') }}</span>
+          </div>
+          {% endif %}
         </div>
+
+        {% if community.metadata.description %}
+          <div class="ui accordion mt-5">
```

Comment:
I would suggest to use something more like "show more/show less" button instead of the accordion, because i am not sure if we want to hide the description entirely

Reply:
So to keep the same as before you mean?